Makefile

```diff
@@ -143,7 +143,8 @@ codegen: $(CODEGEN)
 verify-codegen: codegen
 	git diff --exit-code
 
-verify-catalog: schema-check
+verify-catalog:
+	go test -v ./test/schema/catalog_versions_test.go
```

Comment:
Verifying catalog version consistency is a unique problem for OLM, and does not need to be part of the separate binary.